### DISCUSSION TITLE
How to get device time?

### DISCUSSION THREAD

**Discussion by ghost** [ionic-team/capacitor#5115] [Q&A]
I couldn't find it in docs. How do I get current time of the user device?
Thanks.

**Comment by buhrmi** [comment]
Have you tried `new Date()`?

**Comment by ghost** [comment]
No, I have not. From which library should I import it?

**Comment by buhrmi** [comment]
you don't need to import it

**Comment by ghost** [comment]
OK, gonna give it a try later, as now I am away from the project....
